apps/omg/lib/omg/state.ex

```diff
@@ -36,6 +38,7 @@ defmodule OMG.State do
     GenServer.start_link(__MODULE__, :ok, name: __MODULE__)
   end
 
+  @decorate measure_event()
```

Comment:
sometimes the decorator are above specs, sometimes below...

Reply:
and what do you prefer, before spec or after?

Reply:
before spec